tests/FakeItEasy.Specs/FakeItEasy.Specs.csproj

```diff
@@ -3,19 +3,8 @@
   <PropertyGroup>
     <TargetFrameworks>net461;netcoreapp1.0</TargetFrameworks>
     <AssemblyName>FakeItEasy.Specs</AssemblyName>
-    <CodeAnalysisRuleSet>FakeItEasy.Specs.ruleset</CodeAnalysisRuleSet>

```

Comment:
I don't see where this is included in the new files.

Reply:
Oops! It isn't. I'll restore this line, since this ruleset isn't included in any other project.
I'm surprised that I didn't get any warning without the ruleset, though... I'll check again.

Reply:
Ah, of course, it's because of #1144...

Reply:
I amended the commit

Reply:
Lovely, M. Levesque.